factory-reset/eos-factory-reset

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/python3
+import argparse
+import logging
+import subprocess
+import sys
+
+from systemd import journal
+
+# Options list
+# Each stage has the form (short name, long name, description)
+OPTS = [
+    ('D', 'disable', 'fully remove a scheduled factory reset'),
+]
```

Comment:
Ok, I think I'd add the `-y` or `-f` flags to skip user confirmation, unless the UI for factory reset enables the stages by hand instead of calling this script.

Reply:
I'm re-structured the code a bit to make it super simple to add `-f`, and added it in a separate commit on branch `T24613-force`. But I don't like the idea of having a way for the user to schedule a factory reset without confirmation, so I don't think we should merge that branch.

I believe the UI should not call this script directly, but some API from a library (that would be used by this program as well).